src/components/Process/Header.tsx

```diff
@@ -30,7 +34,16 @@ const ProcessHeader = () => {
   )
   const strategy = useStrategy()
 
+  useEffect(() => {
+    ;(async () => {
+      if (!election?.census?.censusId || !client) return
+      const censusInfo: CensusInfo = await client.fetchCensusInfo(election.census.censusId)
```

Comment:
What happens if the call fails? Shouldn't this maybe be in a try/catch and handle the error somehow?

Reply:
If the call fails the census info it remains just undefined so doesn't affect to the rest of the code. Anyway, i'll set this inside a try catch to avoid console errors

Reply:
d4978e9